lib/stripe/subscriptions/credit_note.ex

```diff
@@ -0,0 +1,188 @@
+defmodule Stripe.CreditNote do
+  @moduledoc """
+  Work with Stripe Credit Note objects.
+
+  You can:
+
+  - Create a credit note
+  - Retrieve a credit note
+  - Update a credit note
+  - Void a credit note
+  - List credit notes
+
+  ```
+  {
+    "id": "ivory-extended-580",
+    "object": "plan",
+    "active": true,
+    "aggregate_usage": null,
+    "amount": 999,
+    "billing_scheme": "per_unit",
+    "created": 1531234812,
+    "currency": "usd",
+    "interval": "month",
+    "interval_count": 1,
+    "livemode": false,
+    "metadata": {
+    },
+    "nickname": null,
+    "product": "prod_DCmtkptv7qHXGE",
+    "tiers": null,
+    "tiers_mode": null,
+    "transform_usage": null,
+    "trial_period_days": null,
+    "usage_type": "licensed"
+  }
+  ```
+  """
+
+  use Stripe.Entity
+  import Stripe.Request
+
+  @type t :: %__MODULE__{
+          id: Stripe.id(),
+          object: String.t(),
+          amount: number,
```

Comment:
should this be `integer` or is `number` valid as well?

Reply:
Yes, good catch :) Pushed a new version